packages/is-valid-cnpj/src/index.ts

```diff
@@ -24,5 +26,8 @@ export default function isValidCnpj(cnpj: string) {
 
   const numericCNPJ = onlyNumbers(cnpj);
 
-  return isValidLength(numericCNPJ) && !belongsToBlacklist(numericCNPJ) && isValidChecksum(numericCNPJ);
+  return isValidCnpjFormat(cnpj)
+    && isValidLength(numericCNPJ)
```

Comment:
You are validate format of cnpj with regex, why do you need isValidLength? Regex not validate also length of cnpj?

Reply:
You're right, not necessary to validate the size, I'll fix it.